process-engine/process_engine_runtime
# ✨ External task topics from token expression

## Changes

1. Allow ExternalTask topics to contain token expressions
2. Add tests for ExternalTasks that use token expressions as a topic

## Issues

Closes #347

PR: #351

## How to test the changes

Best use BPMN Studio and an ExternalTask worker app of your choice.

- Create a ProcessModel with an ExternalTask that has a token expression as a topic (i.e. `token.history.someFlowNodeId.someReference`)
- Create an ExternalTask Worker that listens for the matching topic
- Run that ProcessModel
    - Make sure you pass the correct topic as an initial payload!
- See that the ExternalTask uses the correct topic and gets processed as expected
